docker/Dockerfile.contrib

```diff
@@ -26,23 +26,23 @@ RUN yarn install --network-timeout=${YARN_NETWORK_TIMEOUT}
 RUN yarn run build:full
 RUN yarn install --production --frozen-lockfile
 RUN for i in config core serial shared; do \
-    cd packages/$i && \
-    yarn version --no-git-tag-version --new-version $(yarn versions --json| \
-        jq -r '[.data."@zwave-js/'${i}'"]'[0])-$(git rev-parse --short HEAD) && \
```

Comment:
The reason this is indented is that it's part of the previous command. I think that's a good thing.

Reply:
> The reason this is indented is that it's part of the previous command. I think that's a good thing.

I think, this was by mistake due to auto-save which I seem to have missed.